src/Common/src/TypeSystem/Ecma/MetadataExtensions.cs

```diff
@@ -72,6 +90,25 @@ public static class MetadataExtensions
             return default(CustomAttributeHandle);
         }
 
+        public static IEnumerable<CustomAttributeHandle> GetCustomAttributeHandles(this MetadataReader metadataReader, CustomAttributeHandleCollection customAttributes,
```

Comment:
Makes sense. The two enumeration is kinda silly. Merged the two functions into one.